engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -1138,50 +1138,49 @@ void Myst::o_bookGivePage(uint16 var, const ArgumentsArray &args) {
 	debugC(kDebugScript, "Card Id (Book Cover): %d", cardIdBookCover);
 	debugC(kDebugScript, "SoundId (Add Page): %d", soundIdAddPage);
 
-	// No page or white page
-	if (!_globals.heldPage || _globals.heldPage == 13) {
-		_vm->changeToCard(cardIdBookCover, kTransitionDissolve);
-		return;
-	}
-
 	uint16 bookVar = 101;
 	uint16 mask = 0;
 
 	switch (_globals.heldPage) {
-	case 7:
+	case kNoPage:
+		// fallthrough
```

Comment:
No need to put a `fallthrough` marker if there is no code in the case. It's used to tell the compiler we didn't forget a break. Here it's obvious the fallthrough is intentional.